alignak/objects/command.py

```diff
@@ -66,44 +66,36 @@ class Command(Item):
     """
     __metaclass__ = AutoSlots
 
+    name_property = "command_name"
     my_type = "command"
 
     properties = Item.properties.copy()
     properties.update({
-        'command_name': StringProp(fill_brok=['full_status']),
-        'command_line': StringProp(fill_brok=['full_status']),
-        'poller_tag':   StringProp(default='None'),
-        'reactionner_tag':   StringProp(default='None'),
-        'module_type':  StringProp(default=None),
-        'timeout':      IntegerProp(default=-1),
-        'enable_environment_macros': BoolProp(default=False),
+        'command_name':
+            StringProp(default='_echo', fill_brok=['full_status']),
+        'command_line':
+            StringProp(default='', fill_brok=['full_status']),
+        'poller_tag':
+            StringProp(default='None'),
+        'reactionner_tag':
+            StringProp(default='None'),
+        'module_type':
+            StringProp(default='fork'),
+        'timeout':
+            IntegerProp(default=-1),
+        'enable_environment_macros':
+            BoolProp(default=False),
     })
 
-    def __init__(self, params=None, parsing=True):
+    def __init__(self, params=None, parsing=True, debug=False):
 
         if params is None:
             params = {}
-        super(Command, self).__init__(params, parsing=parsing)
-
-        if not hasattr(self, 'timeout'):
-            self.timeout = -1
-
-        if not hasattr(self, 'poller_tag'):
-            self.poller_tag = 'None'
-        if not hasattr(self, 'enable_environment_macros'):
-            self.enable_environment_macros = False
-        if not hasattr(self, 'reactionner_tag'):
-            self.reactionner_tag = 'None'
-        if not hasattr(self, 'module_type'):
-            # If the command start with a _, set the module_type
-            # as the name of the command, without the _
-            if getattr(self, 'command_line', '').startswith('_'):
-                module_type = getattr(self, 'command_line', '').split(' ')[0]
-                # and we remove the first _
-                self.module_type = module_type[1:]
-            # If no command starting with _, be fork :)
-            else:
-                self.module_type = 'fork'
+        super(Command, self).__init__(params, parsing=parsing, debug=debug)
```

Comment:
It is inheriting from an Item. As such the properties are managed in the Item initialization function